armi/operators/operator.py

```diff
@@ -73,6 +73,10 @@ class Operator:
 
     .. note:: The :doc:`/developer/guide` has some additional narrative on this topic.
 
+    .. impl:: The operator package shall expose an ordered list of interfaces, and loop over them in order.
+        :id: I_ARMI_OPERATOR_INTERFACES2
```

Comment:
I see the `0` but not the `1`, so is this supposed to be `1`?

Reply:
Fixed!